packages/faust-nx/tests/server/auth/middleware.test.ts

```diff
@@ -0,0 +1,136 @@
+import 'isomorphic-fetch';
+import fetchMock from 'fetch-mock';
+import { IncomingMessage, ServerResponse } from 'http';
+import { authorizeHandler } from '../../../src/server/auth/middleware';
+import * as getWpUrl from '../../../src/lib/getWpUrl';
+import * as getWpSecret from '../../../src/lib/getWpSecret';
```

Comment:
We may want to consider implementing a [tsconfig.json](https://www.typescriptlang.org/tsconfig#baseUrl) to clean up the `../../../`

Reply:
I would typically agree but since this is just for tests and not source code, adding a `tsconfig.json` may be more complexity than it's worth.